### ISSUE TITLE
Parse error on open-ended time-range (value ".." (tDOT2))

### ISSUE THREAD

**Issue by fwolfst** [presidentbeef/brakeman#1650]
### Background

Brakeman version: 5.1.1
Rails version: 6.1.3.2
Ruby version: ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-darwin20]

Link to Rails application code: n/a

#### Parse Error

Minimal example that does not parse:

Sorry, idk. In a concern I define a scope containing

```ruby
scope :at_point_in_time, ->(point_in_time) do
  where(starts_at: ..point_in_time, ends_at: point_in_time..)               
    .or(open_ended.where(starts_at: ..point_in_time))
end
```

results in
```
Error: app/models/concerns/xxx.rb:9 :: parse error on value ".." (tDOT2) Could not parse app/models/concerns/xxx.rb
```

`ruby_parse` seems to run fine.

**Comment by presidentbeef** [comment]
Did you try Brakeman 5.1.2?

**Comment by fwolfst** [comment]
Okay, sorry. Fixed in 5.1.2 .
